README.md

```diff
@@ -123,16 +123,31 @@ Merging (needed to set `--signoff` and export environment variables):
 git duet-merge -v [any other git options]
 ```
 
+Rebasing (needed to set `--signoff` and export environment variables):
+
+``` bash
+git duet-rebase [any other git options]
+```
+
+Cherry-picking (needed to set `--signoff` and export environment variables):
+
+``` bash
+git duet-cherry-pick [any other git options]
+```
+
 Suggested aliases:
 
 ```
 dci = duet-commit
 drv = duet-revert
 dmg = duet-merge
+drb = duet-merge
```

Comment:
I just cloned it down from your repo on master, commit c21558e3634f657cab36752c69dc5800c8a8236a

It shows:

```
dmg = duet-merge
drb = duet-merge
dcp = duet-cherry-pick
```